src/containers/Navbar/Navbar.container.tsx

```
@@ -13,6 +21,89 @@ import { LeftWrapper, StyledAppBar, StyledBox } from './Navbar.styles';
+
+    /**
+     * Users to be displayed in the search box component that is fetched from the API.
+     */
+    const [searchUsers, { data: users, isFetching: isFetchingUsers }] =
+        useLazySearchUsersQuery();
+
+    /**
+     * Watches for changes in the user input and triggers a search for users with the current input value.
+     */
+    useEffect(() => {
+        if (searchUser && searchUser.userName) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)